### PR TITLE
Doppler kubernetes operator

### PR DESCRIPTION
**Pull request by sergio-chainguard**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: doppler-kubernetes-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package:
+  name: doppler-kubernetes-operator
+  version: 1.5.0
+  epoch: 0
+  description: ""
```
Comment (jamonation) [inline doppler-kubernetes-operator.yaml:5]: A description would be good here, took me a bit to figure out what this is for

File: doppler-kubernetes-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      deps: github.com/gogo/protobuf@v1.3.2 golang.org/x/crypto@v0.17.0 golang.org/x/net@v0.17.0 github.com/prometheus/client_golang@v1.11.1
+      go-version: "1.21"
+
+  - runs: |
+      set -x
+      CGO_ENABLED=0 GO111MODULE=on GOOS=$(go env GOOS) GOARCH=$(go env GOARCH)
+      # https://github.com/DopplerHQ/kubernetes-operator/blob/a147cf3dbdf5ef517a73953e8d1f71baeafc4f24/Dockerfile#L23C72-L23C211
+      go build -a --ldflags="-X \
+      github.com/DopplerHQ/kubernetes-operator/pkg/version.ControllerVersion=${{package.version}}" \
+      -o manager main.go
```
Comment (Dentrax) [inline doppler-kubernetes-operator.yaml:33]: we can use `go/build` pipeline here. example: https://github.com/mauricio-dc-chainguard/os/blob/b3ab2e6a64771e1ebc7e3f23b4069dd91bbbeac7/kuberay-operator.yaml#L24-L29

File: doppler-kubernetes-operator.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      tag: "v${{package.version}}"
+      expected-commit: a147cf3dbdf5ef517a73953e8d1f71baeafc4f24
+
+  - uses: go/bump
+    with:
+      deps: github.com/gogo/protobuf@v1.3.2 golang.org/x/crypto@v0.17.0 golang.org/x/net@v0.17.0 github.com/prometheus/client_golang@v1.11.1
+      go-version: "1.21"
+
+  - runs: |
+      set -x
```
Comment (Dentrax) [inline doppler-kubernetes-operator.yaml:28]: we shouldn't need this

File: doppler-kubernetes-operator.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  github:
+    identifier: DopplerHQ/kubernetes-operator
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - wolfi-base
+        - doppler-kubernetes-operator
```
Comment (ajayk) [inline doppler-kubernetes-operator.yaml:48]:
```suggestion
```

**Review by ajayk — APPROVED** [review]

**Pull request merged**